ci/playbooks/pre-commit.yml

```diff
@@ -34,3 +40,12 @@
             target: pre_commit_nodeps
             params:
               USE_VENV: "no"
+              LOG_DIR: "{{ ansible_user_dir }}/zuul-output/logs"
+      always:
+        - name: Expose pre-commit log as artifact
+          zuul_return:
+            data:
+              zuul:
+                artifacts:
+                  - name: pre-commit logfile
+                    url: pre_commit.log
```

Comment:
will you expand that to add the others? check_zuul_files.log etc

Reply:
once we get more of those tests, yes. Have to consider how to do it "right" to avoid code duplication though. Maybe same playbook, with some "when" and appropriate parameter passed down the zuul job? Not sure yet, so I'd just go with that pre-commit alone for now.

Reply:
FYI, here's what I'm thinking for adding the other jobs: #1584 
We'll need another one to start getting job dependency via #1573 - but things are coming nicely.